src/library_signals.js

```diff
@@ -116,34 +64,14 @@ var funs = {
       if (__sigalrm_handler) {{{ makeDynCall('vi', '__sigalrm_handler') }}}(0);
     }, seconds*1000);
   },
-  ualarm: function() {
```

Comment:
I don't see ualarm added from musl code?

Reply:
We were already including musl's `ualarm` since it lives in `unistd/ualarm.c` which was already being compiled in.